.github/workflows/pytest.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Set env context
         run: |
           echo CONDA_PATH=$CONDA >>$GITHUB_ENV
-          echo RUN_STAMP=${{ runner.os }}-${{ (env.cuda_compiler_version != 'None') && format('cuda{0}', env.cuda_compiler_version) || 'cpu' }} >>$GITHUB_ENV
+          echo RUN_STAMP=${{ runner.os }}-cpu >>$GITHUB_ENV
```

Comment:
both pytest.yml and modin.yml should be made a part of test.yml, thus the whole "Set env context" section will be reusable

Reply:
will change it later in further prs